week06/manager_test.go

```diff
@@ -0,0 +1,172 @@
+package week06
+
+import (
+	"testing"
+)
+
+func TestManagerStart(t *testing.T) {
+	t.Parallel()
+	testCases := []struct {
+		name	string
+		emp 	int
+		err 	error
+	}{
+		{name: "start manager with invalid employee number", emp: -1, err: ErrInvalidEmployeeCount(-1)},
+		{name: "start manager with valid employee number", emp: 2, err: nil},
+	}
+	for _, tc := range testCases {
+		t.Run(tc.name, func(t *testing.T) {
+
+			m := NewManager()
+			act := m.Start(tc.emp)
```

Comment:
You need to clean up your resources.

```suggestion
			act := m.Start(tc.emp)
			defer m.Stop()
```